src/Symfony/Component/Mailer/Bridge/Amazon/Transport/SesApiTransport.php

```diff
@@ -130,4 +131,19 @@ private function getPayload(Email $email, Envelope $envelope): array
 
         return $payload;
     }
+
+    protected function stringifyAddresses(array $addresses): array
```

Comment:
Does it make sense to move this method to a trait or to a separate service to avoid code duplication?

Reply:
I have no strong opinions about that. You could.. but it is just two places.. but there are a few lines of logics... 

¯\_(ツ)_/¯

Reply:
I choose to not use a trait because one of these classes is deprecated when code will be removed in 6.0 we will have a technical dept with a trait used by a single class.